platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/reports/FatalIssueReporter.kt

```diff
@@ -150,7 +154,9 @@ internal class FatalIssueReporter(
                                 FatalIssueReporterStorage(destinationDirectory.destinationDirectory),
                             )
                         captureUncaughtExceptionHandler.install(this)
-                        persistLastExitReasonIfNeeded(appContext)
+                        backgroundThreadHandler.runAsync {
```

Comment:
any concerns about this causing a race condition for the appexitinfo case between `initFatalIssueReporting()` and `start()` (which would read the file report) and now  `processFatalIssue()`?